CMakeLists.txt

```diff
@@ -552,7 +559,7 @@ endif(CCACHE)
 
 # Check for external dependencies ==============================================
 include(external/pthread.cmake)
-include(external/mpi.cmake)
+#include(external/mpi.cmake)
```

Comment:
was this meant to be commented out?

Reply:
Good catch ... on my machine mpi.cmake just gets stuck in an infinite (or at least 5min+) long loop so I just commented it out --- everything seems to work fine without it.

I'll repush with it uncommented in case it is needed on other platforms.